health/micro-ui/web/micro-ui-internals/packages/modules/health-payments/src/pages/employee/project_selection.js

```
@@ -0,0 +1,181 @@
+                </Card>
+            </div>
+            <ActionBar
+                actionFields={[
+                    <Button
+                        icon="ArrowBack"
+                        label={t("HCM_AM_BACK_LABEL")}
+                        onClick={() => history.push(`/${window.contextPath}/employee`)}
+                        style={{ marginLeft: "2.5rem", minWidth: "14rem" }}
+                        type="button"
+                        variation="secondary"
+                    />,
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)